fixtures/v2/api/deleteRegistrantChange/success.http

```diff
@@ -1,4 +1,4 @@
-HTTP/1.1 201
+HTTP/1.1 204 No Content
 server: nginx
 date: Tue, 22 Aug 2023 11:14:44 GMT
 content-type: application/json; charset=utf-8

```

Comment:
Ideally, this should still be omitted, as the content is not JSON (an empty string is not valid JSON).

Reply:
Agree, would you mind opening a separate issue in the API backend repo to track this change we want to make?